modules/account-lib/test/unit/mpc/curves/secp256k1.ts

```diff
@@ -0,0 +1,106 @@
+    valueOne.should.equal(testValue - BigInt(1));
+    const valueTwo = sec256k1.scalarAdd(BigInt(5), testValue);
+    valueTwo.should.equal(BigInt(5) + testValue);
+  });
+
+  it('should correctly perform scalar sub', function () {
+    const valueOne = sec256k1.scalarSub(sec256k1.order() + BigInt(2) * testValue, testValue);
+    valueOne.should.equal(testValue);
+    const valueTwo = sec256k1.scalarSub(BigInt(5), testValue);
+    valueTwo.should.equal(sec256k1.order() + (BigInt(5) - testValue));
+  });
+
+  it('should correctly perform scalar multiplication', function () {
+    const valueOne = sec256k1.scalarMult(sec256k1.order(), testValue);
+    valueOne.should.equal(BigInt(0));
+    const valueTwo = sec256k1.scalarMult(BigInt(5), testValue);
+    valueTwo.should.equal(BigInt(5) * testValue);
+  });
+
+  it('should correctly perform scalar reduce', function () {
+    const valueOne = sec256k1.scalarReduce(sec256k1.order() + testValue);
+    valueOne.should.equal(testValue);
+    const valueTwo = sec256k1.scalarReduce(sec256k1.order() - testValue);
+    valueTwo.should.equal(sec256k1.order() - testValue);
+  });
+
+  it('should correctly perform scalar negate', function () {
+    const valueOne = sec256k1.scalarNegate(sec256k1.order() + testValue);
+    valueOne.should.equal(sec256k1.order() - testValue);
+    const valueTwo = sec256k1.scalarNegate(sec256k1.order() - testValue);
+    valueTwo.should.equal(testValue);
+  });
+
+  it('should correctly perform inverts', function () {
+    // Had to convert bigint to string as comparison was not properly working in mocha
+    const testOne = sec256k1.scalarInvert(BigInt(3)).toString();
+    testOne.should.equal('77194726158210796949047323339125271901891709519383269588403442094345440996225');
+
+    const testTwo = sec256k1.scalarInvert(sec256k1.order() + BigInt(1)).toString();
+    testTwo.should.equal('1');
+
+    const testThree = sec256k1
+      .scalarInvert(BigInt('57657865876576467547584635432343132435146576543543134312435443514313234131324551034534'))
+      .toString();
+    testThree.should.equal('45199239020920791752653273479365580486664737121914751844510223387235717825519');
+
+    const testFour = sec256k1.scalarInvert(BigInt('-23232424342224')).toString();
+    testFour.should.equal('109007816100548644677753129517258866095932985532097924870123596204155595187610');
+
+    const testFive = sec256k1.scalarInvert(sec256k1.order() + testValue).toString();
+    testFive.should.equal('16664411509403755791375590925833668071820872792496863322396593062116822075060');
+  });
+
+  it('should correctly perform point add', function () {
+    const A = BigInt('0x0279be667ef9dcbbac55a06295ce870b07029bfcdb2dce28d959f2815b16f81798');
+    const B = BigInt('0x0252972572d465d016d4c501887b8df303eee3ed602c056b1eb09260dfa0da0ab2');
+    const publicKey = sec256k1.pointAdd(A, B).toString();
+    publicKey.should.equal('449973406896538015988913407864183977074571439549869550234638933112191976505998');
```

Comment:
Let's specify this result as hex, since it's really not a number, but a compact byte-encoding of an x-y point.